capella2polarion/data_model/work_item_attachments.py

```diff
@@ -116,9 +118,15 @@ def content_checksum(self) -> str:
             try:
                 elk_input = self.diagram.elk_input_data(self.render_params)
                 if isinstance(elk_input, tuple):
-                    input_str = ";".join(eit.json() for eit in elk_input)
+                    input_str = ";".join(
+                        remove_width_and_height(eit).model_dump_json()
+                        for eit in elk_input
+                    )
```

Comment:
This should not be necessary and does not really look clean to me. You could argue that mocking the elk_input instead is fine as it should be tested in context_diagrams, but I would prefer to work with real input here to ensure that we know, if an update of context diagrams changes the rendering of diagrams

Reply:
The `width` and `height` calculations depend from the used platform (MacOS, Linux or Windows). They are not important for identifying if the context-diagram changed, the collected input elements are reliable and give developers the freedom of choice for their used platform without broken tests.